tests/unit/test_venv.py

```diff
@@ -0,0 +1,29 @@
+import tomllib
+
+MAIN_PYPROJECT = "pyproject.toml"
+MOE_PYPROJECT = "deps/moe/pyproject.toml"
+
+
```

Comment:
maybe add comment why we need this tests

Reply:
also add instruction for what to do when this test is failing

Reply:
doone